lib/BinaryNode.cpp

```diff
@@ -39,22 +39,20 @@ BinaryNode::BinaryNode(void* p, qore_size_t size) : SimpleValueQoreNode(NT_BINAR
 }
```

Comment:
BinaryNode ctor takes ownership of `p` (I would state that in documentation comment) and `p` must be allocated via malloc only